io.github.pastthepixels.InfiniteShooter.yml

```diff
@@ -0,0 +1,32 @@
+app-id: io.github.pastthepixels.InfiniteShooter
+runtime: org.freedesktop.Platform
+runtime-version: 21.08
+sdk: org.freedesktop.Sdk
+command: InfiniteShooter.x86_64
+finish-args:
+  - --share=ipc
+  - --socket=x11 # <-- Godot needs X11
+  - --socket=pulseaudio
+  - --device=all # <-- This flag enables controller support
+modules:
+  # InfiniteShooter (the game)
+  - name: InfiniteShooter
+    buildsystem: simple
+    build-commands:
+      - install -D InfiniteShooter-LINUX.x86_64 /app/bin/InfiniteShooter.x86_64
+    sources:
+      - type: file
+        url: https://github.com/pastthepixels/InfiniteShooter/releases/download/v0.4.0/InfiniteShooter-LINUX.x86_64 # <-- Edit with latest verion
```

Comment:
this is x86_64 only but Flathub still try to build an aarch64 flatpak.

1. either you provide a aarch64 build
2. or you instruct Flathub to only build x86_64 https://github.com/flathub/flathub/wiki/App-Maintenance#only-build-on-x86_64

Reply:
Oh, right! Sorry about that! I'll try to get something working with org.godotengine.godot.BaseApp so that I can provide aarch64 builds.